Calendar/CalLogHelpers/Build-CalLogCSV.ps1

```diff
@@ -0,0 +1,266 @@
+﻿# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+# Build-CalLogCSV.ps1
```

Comment:
This doesn't need to change, but `Build-CalLogCSV` isn't a function within this file. If it is just a collection of functions, a short name + functions is what we typically do. 

Example: https://github.com/microsoft/CSS-Exchange/blob/main/Shared/LoggerFunctions.ps1

Not saying this is what you need to do, but just something to consider.

Reply:
Done.   Thanks.  That is better than forcing the File names into the Verb-Nown  of PS actions.